ava-test/which.js

```diff
@@ -0,0 +1,47 @@
+import fs from 'fs';
+
+import test from 'ava';
+
+import shell from '..';
+
+shell.config.silent = true;
+
+//
+// Invalids
+//
+
+test('no args', t => {
+  shell.which();
+  t.truthy(shell.error());
+});
+
+test('command does not exist in the path', t => {
+  const result = shell.which('asdfasdfasdfasdfasdf'); // what are the odds...
+  t.falsy(shell.error());
+  t.falsy(result);
+});
+
+//
+// Valids
+//
+
+// TODO(nate): make sure this does not have a false negative if 'git' is missing
+test('basic usage', t => {
+  const git = shell.which('git');
```

Comment:
Why not use `node`? Surely if this test is running on a system, that system has `node` installed...